js/script.js

```diff
@@ -99,13 +169,65 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
   const title = document.querySelector('#title').value;
   const author = document.querySelector('#author').value;
   const isbn = document.querySelector('#isbn').value;
+  const email = document.querySelector('#authorEmail').value;
+  const phone = document.querySelector('#author-phone').value;
```

Comment:
You can extract this into another method to keep this clean. The method could return an object with all the fields populated.
```suggestion
  const book = getFormValues();
```